src/mumble/GlobalShortcut.cpp

```diff
@@ -849,6 +849,9 @@ bool GlobalShortcutEngine::handleButton(const QVariant &button, bool down) {
 
 	if (down) {
 		AudioInputPtr ai = g.ai;
+		if (ai->activityState == AudioInput::ActivityStateIdle) {
+			ai->activityState = AudioInput::ActivityStateExitedIdle;
```

Comment:
This is a data race. However, so is accessing ai->tIdle from the main thread like we do below.
Maybe add a comment:

    // XXX: This is a data race: we write to ai->activityState (accessed by the AudioInput thread) from the main thread.

?

Alternatively, we should fix it by putting both of these accesses behind a lock. They're related, so it shouldn't be bad to do. But we can do it in a different PR.